README.md

```diff
@@ -206,10 +212,11 @@ DataBus::LegState legState=DataBus::RS;                //初始腾空腿
 }
 ```
 
-## 联系方式
+## Contact Information
+
+Developers are welcome to contribute to the optimization and improvement of this code repository!
 
-欢迎各位开发者参与本代码库的优化与提高！
+[💬 Start a Discussion](https://atomgit.com/openloong/openloong-dyn-control/discussions/new/choose) | [📝 Report an Issue](https://atomgit.com/openloong/openloong-dyn-control/issues/create) | [📨 [Submit a Change Request](https://atomgit.com/openloong/openloong-dyn-control/changes)](https://atomgit.com/openloong/openloong-dyn-control/changes)
```

Comment:
please use the github discussion/issue/pull request link instead of atomgit links